app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.health = 100
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    @classmethod
+    def __str__(cls) -> str:
```

Comment:
The `__str__` method is typically an instance method, not a class method. It should be defined without the `@classmethod` decorator and should take `self` as the first parameter instead of `cls`. This will allow it to return a string representation of an individual instance rather than the class.